backend/core/src/shared/filter/custom_filters.ts

```diff
@@ -5,16 +5,25 @@ import {
 } from "../../listings/types/listing-filter-keys-enum"
 import { filterTypeToFieldMap } from "../../listings/dto/listing.dto"
 
-export function addSeniorHousingQuery(qb: WhereExpression, filterValue: string) {
+export function addSeniorHousingQuery(
+  qb: WhereExpression,
+  filterValue: string,
+  includeNulls?: boolean
+) {
   const whereParameterName = ListingFilterKeys.seniorHousing
   const seniorHousingCommunityType = "senior62"
   const reservedCommunityTypeColumnName = `LOWER(CAST(${
     filterTypeToFieldMap[ListingFilterKeys.seniorHousing]
   } as text))`
   if (filterValue == "true") {
-    qb.andWhere(`${reservedCommunityTypeColumnName} = LOWER(:${whereParameterName})`, {
-      [whereParameterName]: seniorHousingCommunityType,
-    })
+    qb.andWhere(
+      `(${reservedCommunityTypeColumnName} = LOWER(:${whereParameterName})  ${
+        includeNulls ? `OR ${reservedCommunityTypeColumnName} IS NULL` : ""
```

Comment:
very nit: Here and elsewhere, can we move the extra space inside the extra string, right before the OR